coral/src/domain/team/team-transformer.ts

```diff
@@ -0,0 +1,7 @@
+import { Team, TeamNamesGetResponse } from "src/domain/team/team-types";
+
+function transformTeamNamesGetResponse(response: TeamNamesGetResponse): Team[] {
```

Comment:
👏  I think it's a really good idea to remove that from the response and leave the responsibility to the component state!